src/ui/components/Scanner/Scanner.tsx

```diff
@@ -187,12 +172,15 @@ const Scanner = forwardRef(
         }
       } catch (e) {
         showError("Scanner Error:", e, dispatch, ToastMsgType.SCANNER_ERROR);
-        initScan();
+        await new Promise((resolve) => setTimeout(resolve, 500)); // Otherwise spam scans
```

Comment:
@sdisalvo-crd I had this in bolnisi too - when restarting the camera you need to put some wait before doing it immediately or you will have a spam of camera scans or restarts. I could see in the logs the scanner error appearing like 10 times very quickly. Need time for user to pull away. 500ms isn't that long for a user so think this is OK?